src/lib/translators/mongo.ts

```diff
@@ -79,6 +79,19 @@ function transformReplace(step: ReplaceStep): MongoStep {
   };
 }
 
+/** transform a 'sort' step into corresponding mongo steps */
+function transformSort(step: SortStep): MongoStep {
+  let sortMongo: PropMap<number> = {};
+  if (step.order === undefined) {
+    step.order = Array(step.columns.length).fill('asc');
```

Comment:
I think we should not mutate `step` in order to avoid side effects. Maybe should we simply do something like:

```javascript
const sortOrders = step.order === undefined ? Array(step.columns.length).fill('asc')  : step.order;
# ...
```